src/gsy_e/setup/kpi/d3asim_2103_2_sided_pab.py

```diff
@@ -13,8 +13,8 @@ def get_setup(config):
 
     ConstSettings.GeneralSettings.DEFAULT_UPDATE_INTERVAL = 1
     ConstSettings.MASettings.MARKET_TYPE = 2
-    ConstSettings.MASettings.MIN_BID_AGE = 0
-    ConstSettings.MASettings.MIN_OFFER_AGE = 0
+    ConstSettings.MASettings.MIN_BID_AGE = 1
+    ConstSettings.MASettings.MIN_OFFER_AGE = 1
```

Comment:
Both actually. The previous implementation had these parameters set to 0 in order to emulate immediate broadcast of the residual bids and offers to all markets. This behavior is achieved by the fix in the Market Agents now though, and setting these parameters to 0 will force the PVs to sell all their energy to the DSO as soon as each house market is cleared. This will affect the intended self sufficiency / consumption values of the setup file, because they both have the intent to first cover the supply / demand of the community / houses and then to use the DSO (that's what the test intends at least). Therefore in order to maintain the intended behavior we will have to set the forwarding age to the default values and not to broadcast them immediately, because otherwise the DSO will buy / sell energy on every House, before other houses are able to clear. 
In case the above is not that clear let me know to discuss this offline with a demo :)

Reply:
Thanks a lot! no, it makes total sense now.